app/src/main/java/org/piramalswasthya/cho/ui/home/OfflineDataSharingBottomSheet.kt

```
@@ -0,0 +1,579 @@
+    private val payloadCallback = object : PayloadCallback() {
+        override fun onPayloadReceived(p0: String, p1: Payload) {
+
+        }
+
+        override fun onPayloadTransferUpdate(endpointId: String, update: PayloadTransferUpdate) {
+            if (update.status == PayloadTransferUpdate.Status.IN_PROGRESS) {
+                val progress = (update.bytesTransferred * 100 / update.totalBytes).toInt()
+                // updateProgress(progress)
+            } else if (update.status == PayloadTransferUpdate.Status.SUCCESS) {
+
+            } else if (update.status == PayloadTransferUpdate.Status.FAILURE) {
```

detekt

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyIfBlock)